### PR TITLE
we no recurse

### PR DESCRIPTION
**Pull request by Porcupinine**
not recursive exponential \0/

### CODE REVIEW

File: tests/tests.cpp
Comment (Esiwehl) [inline tests/tests.cpp:1]: I digs, approved

File: .gitignore
Comment (Esiwehl) [inline .gitignore:1]: hmm, why?

File: include/example.h
Comment (Esiwehl) [inline include/example.h:1]: yesyes, very necessary

**Review by dmaessen — APPROVED** [review]
nice! maybe also good to test with negatives

**Review by Esiwehl — APPROVED** [review]

**Pull request merged**